### PR TITLE
Logging build version and commit hash of operator and sidecar jars

### PR DESCRIPTION
**Pull request by benbromhead**
To address issue #85 and make issue diagnosis a little easier (what version are you using)?

### CODE REVIEW

**Comment by zegelin** [comment]
Can we add some of this version info to `https://github.com/instaclustr/cassandra-operator/blob/master/java/common/src/main/java/com/instaclustr/picocli/ManifestVersionProvider.java`?  That way `--version` on the CLI will also do the right thing.

**Comment by benbromhead** [comment]
Added

**Comment by benbromhead** [comment]
Updated to address comments

File: java/common/src/main/java/com/instaclustr/picocli/GitPropertiesVersionProvider.java [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+
+
     @Override
     public String[] getVersion() {
         String version = "Unknown version";
         try {
             version = Info.getInfoString();
         } catch (IOException e) {
-            e.printStackTrace();
+            logger.warn("Could not detect version");
```
[the remark this answers is not in the document]
Comment (benbromhead) [reply java/common/src/main/java/com/instaclustr/picocli/GitPropertiesVersionProvider.java]: Fixed

**Pull request merged**